### DISCUSSION TITLE
Counter Metric Support

### DISCUSSION THREAD

**Discussion by sky93** [spatie/laravel-prometheus#20] [Ideas]
**Description**
The current implementation of `spatie/laravel-prometheus` includes support for Gauge metrics, as exemplified by the `Prometheus::addGauge` method. However, there is no equivalent support for Counter metrics. 

**Use Case**
This feature would be beneficial for scenarios where it is necessary to count occurrences of certain events, rather than measuring their current value or state, which is what Gauge metrics do.

**Potential Implementation**
A possible implementation approach could be similar to how Gauges are handled but tailored for the specific nature of Counter metrics.

I believe this addition would enhance the versatility of the package and provide users with a more comprehensive set of tools for Prometheus metrics in Laravel.

**Comment by freekmurze** [comment]
Feel free to send a PR
